packages/expo-cli/scripts/introspect.ts

```diff
@@ -319,8 +317,8 @@ if (['markdown', 'md'].includes(process.argv[2])) {
     const booleanArg: FigArg = {
       name: 'boolean',
       suggestions: [
-        // { name: 'true', icon: ICON.true },
-        // { name: 'false', icon: ICON.false },
+        { name: 'true', icon: ICON.true },
+        { name: 'false', icon: ICON.false },
```

Comment:
Was is commented because of some particular reason?

Reply:
yes, we don't support `expo start --dev-client true` originally fig didn't show the autocomplete to true/false so we added them, but the change happened so they became commented.